docs/tutorials/uniq-factories/creating-uniq-factories/how-to-create-uniq-metadata.md

```diff
@@ -0,0 +1,438 @@
+---
+title: 'How to create Uniq Metadata'
+order: 2
+outline: [0, 4]
+---
+
+# How to create Uniq Metadata
+
+Uniqs are tokens that can be associated with anything; games, assets, keys, items, and much more. The data that represents those things _is the metadata_. The title, the descriptions, and the images are all metadata.
+
+Ultra has a specific format for how we expect metadata to be organized.
+
+## Prerequisites
+
+-   A basic understanding of [Factory Metadata](../uniq-variants/factory-metadata.md), [Default Uniq Metadata](../uniq-variants/uniq-default-metadata.md) and [Uniq Metadata](../uniq-variants/uniq-metadata.md).
+-   You will need to access, and make a copy of the [Metadata Template Google Sheet](https://docs.google.com/spreadsheets/d/1Gi0iuJis-riKkyhYgMRYVnhGD6PbjvBv8U7lhRwdhNk/edit?usp=sharing) in your own Google account.
```

Comment:
Guide already explains what you need to do to copy and use the template. Maybe just replace this with "You will need a Google account to make a copy of the Metadata Template Google Sheet"